reference_algorithms/target_setting_algorithms/ogbg/tuning_search_space.json

```diff
@@ -1,12 +1,12 @@
 {
     "learning_rate": {
         "feasible_points": [
-            1.60808
+            2.4917728606918423
         ]
     },
     "beta1": {
         "feasible_points": [
-            0.9536
+            0.9449369031171744
         ]
     },
     "warmup_steps": {
```

Comment:
decay_steps_factor should be `0.861509027839639`

Reply:
Thanks for catching this